StRoot/StRefMultCorr/StRefMultCorr.cxx

```diff
@@ -80,1245 +80,1157 @@
+  Double_t correction_luminosity;
+  if( mParameterIndex>=36 && mParameterIndex<=37 ) {
+    // if(mYear[mParameterIndex] == 2018 && IsZr) zdcmean = 96.9914;
+    // if(mYear[mParameterIndex] == 2018 && IsRu) zdcmean = 97.9927;
+    Double_t b_prime = 1.;
+    if(mParameterIndex==36) b_prime = 96.9914; // Zr
+    if(mParameterIndex==37) b_prime = 97.9927; // Ru
+    correction_luminosity = (par0_lum==0.000) ? 1.0 : b_prime/(par0_lum+zdcCoincidenceRate*par1_lum);
+  }
+  else {
+    correction_luminosity = (par0_lum==0.000) ? 1.0 : 1.0/(1.0 + par1_lum/par0_lum*zdcCoincidenceRate/1000.);
+  }
+
+  // from Run14, P16id, for VpdMB5/VPDMB30/VPDMB-noVtx, use refMult at ZdcX=30, other is at ZdcX=0;  
+  // -->changed by xlchen@lbl.gov, Run16 ~ 50kHz
+  if( 
+     ( mSubName.CompareTo("Run14_AuAu200_VpdMB5", TString::kIgnoreCase) == 0 && mLibName.CompareTo("P16id", TString::kIgnoreCase) == 0 )
+     || ( mSubName.CompareTo("Run14_AuAu200_VpdMB30", TString::kIgnoreCase) == 0 && mLibName.CompareTo("P16id", TString::kIgnoreCase) == 0 )
+     || ( mSubName.CompareTo("Run14_AuAu200_VpdMBnoVtx_LowMid", TString::kIgnoreCase) == 0 && mLibName.CompareTo("P16id", TString::kIgnoreCase) == 0 )
+     || ( mSubName.CompareTo("Run14_AuAu200_VpdMBnoVtx_High", TString::kIgnoreCase) == 0 && mLibName.CompareTo("P15ic", TString::kIgnoreCase) == 0 )
+     || ( mSubName.CompareTo("Run16_AuAu200_VpdMB5", TString::kIgnoreCase) == 0 && mLibName.CompareTo("P16ij", TString::kIgnoreCase) == 0 )
+     || ( mSubName.CompareTo("Run16_AuAu200_VpdMBnoVtx", TString::kIgnoreCase) == 0 && mLibName.CompareTo("P16ij", TString::kIgnoreCase) == 0 )
+      ) {
+    float zdcmean = 0;
+    if(mYear[mParameterIndex] == 2014) zdcmean = 30.;
+    if(mYear[mParameterIndex] == 2016) zdcmean = 50.;
+    correction_luminosity = (par0_lum==0.0) ? correction_luminosity : correction_luminosity*(par0_lum+par1_lum*zdcmean)/par0_lum; 
+  }
+
+  // par0 to par5 define the parameters of a polynomial to parametrize z_vertex dependence of RefMult
+  const Double_t par0 = mPar_z_vertex[0][mParameterIndex];
+  const Double_t par1 = mPar_z_vertex[1][mParameterIndex];
+  const Double_t par2 = mPar_z_vertex[2][mParameterIndex];
+  const Double_t par3 = mPar_z_vertex[3][mParameterIndex];
+  const Double_t par4 = mPar_z_vertex[4][mParameterIndex];
+  const Double_t par5 = mPar_z_vertex[5][mParameterIndex];
+  const Double_t par6 = mPar_z_vertex[6][mParameterIndex];
+  const Double_t par7 = mPar_z_vertex[7][mParameterIndex]; // this parameter is usually 0, it takes care for an additional efficiency, usually difference between phase A and phase B parameter 0
+
+  const Double_t  RefMult_ref = par0; // Reference mean RefMult at z=0
+  const Double_t  RefMult_z   = par0 + par1*z + par2*z*z + par3*z*z*z + par4*z*z*z*z + par5*z*z*z*z*z + par6*z*z*z*z*z*z; // Parametrization of mean RefMult vs. z_vertex position
+  Double_t  Corr_z = 1.0; // Correction factor for RefMult, takes into account z_vertex dependence
```

Comment:
I appreciate your effort to give meaningful names to local variables but aren't we supposed to follow the STAR coding guidelines?

> Variable names follow camel case convention and start with a lower case letter: myLocalVariable.

https://www.star.bnl.gov/public/comp/sofi/guidelines/formatguide.xml#Variable_Names

Reply:
Those parameters do not have any meaningful names since those are the coefficients of the 6th-order polynomial. 
As for the camel case, I'll do it at some point when will have more time.